server/src/main/java/org/apache/druid/server/ClientInfoResource.java

```diff
@@ -300,6 +306,7 @@ public Iterable<LocatedSegmentDescriptor> getQueryTargets(
       @PathParam("dataSourceName") String datasource,
       @QueryParam("intervals") String intervals,
       @QueryParam("numCandidates") @DefaultValue("-1") int numCandidates,
+      @QueryParam("cloneQueryMode") CloneQueryMode cloneQueryMode,
```

Comment:
Use String param and `QueryContexts.getAsEnum()`, same as suggested in the other resource class.